deps/llhttp/src/llhttp.c

```diff
@@ -6037,6 +6037,7 @@ int llhttp__internal_execute(llhttp__internal_t* state, const char* p, const cha
     if (error != 0) {
       state->error = error;
       state->error_pos = endp;
+      return error;
```

Comment:
Maybe a unit test?

Reply:
It is covered by https://github.com/indutny/llparse/commit/f6eea30c4fa853a59580e6a7056512c08ff34f15#diff-aa87b56401c951fd0ce18bd3cd5a4e67